### DISCUSSION TITLE
Zarchive for CHD as compression for data

### DISCUSSION THREAD

**Discussion by Papermanzero** [mamedev/discussions#96] [Ideas]
The idea is to use Zarchive from excap as data compression for the next version of CHD. 
https://github.com/Exzap/ZArchive/releases

Zarchive provides currently the best way to compress data especially for modern disc based games (wiiU, xbox, 360, ps3) 

moreover zarchive is a lossless format which is perfect for archiving original discs.

Zarchive would provide a benefit compared to the current lzma or zlib algorithms.
https://github.com/Exzap/ZArchive

**Comment by rb6502** [comment]
CHDMAN already supports Zstd compression, it was added in the 0.262 release.

**Reply by Papermanzero** [reply]
But is zstd not different compared to zarchive?
Just asking because several emulator projects discuss about zarchive vs chd. But i think if chd supports the compression from zarchive it would solve everything. Personally i think chd is the future for all emulating projects.

if chd is already supporting it would be an awesome news for many projects

Edit: thanks for the support! Yes I read it is the same. Thanks a lot!

**Reply by rb6502** [reply]
Zstd is what Zarchive uses, yes.
